### ISSUE TITLE
IOS: Share video

### ISSUE THREAD

**Issue by nguyen95** [Lg0gs/react-native-tiktok#11]
I use Share function to share video to tiktok but tiktok app open only, not navigate to post screen. I do this with local file path like var/...../video.mp4. And my app in Under review

**Comment by nguyen95** [comment]
@Lg0gs Can  you support me for this issue?

**Comment by Lg0gs** [comment]
@nguyen95 
Do you mean that you submitted the App to the Tiktok developers portal and is still under review?

**Comment by nguyen95** [comment]
Yes, and now it was rejected. Share sdk only work if my app was approved, right?

**Comment by Lg0gs** [comment]
@nguyen95 
Yes, without it you can't login or even share since you need tiktok app key and secret

**Comment by nguyen95** [comment]
Clearly. Thank you
